Client/MoonlightClient.cpp

```diff
@@ -57,6 +56,9 @@ int MoonlightClient::StartStreaming(std::shared_ptr<DX::DeviceResources> res,Str
 	sprintf(message, "Inserted App ID %d\n", sConfig->appID);
 	Utils::Log(message);
 	int a = gs_start_app(&serverData, &config, sConfig->appID, false, true, 0);
+	if (a != 0) {
+		Utils::Log("Failed to start app");
+	}
 	CONNECTION_LISTENER_CALLBACKS callbacks;
```

Comment:
You should call `LiInitializeConnectionCallbacks()` to zero these prior to use, otherwise any new field/callback added will be left uninitialized.

There are similar functions for `STREAM_CONFIGURATION`, `SERVER_INFORMATION`, `DECODER_RENDERER_CALLBACKS`, and `AUDIO_RENDERER_CALLBACKS` which you should also use.